nemo/backends/pytorch/actions.py

```diff
@@ -951,11 +951,13 @@ def __extract_dynamic_axes(port_name: str, ntype: NeuralType, dynamic_axes: defa
         # for input_ports
         for port_name, ntype in module.input_ports.items():
             if port_name in inputs_to_drop:
+                input_names.remove(port_name)
```

Comment:
Other actions.py changes from #473 are missing.

Reply:
Ok, I do not know #473 but some things are now clearly redundant, at least when it comes to jasper deployment...